stable/traefik/README.md

```diff
@@ -114,11 +114,26 @@ $ helm install --name my-release --namespace kube-system --values values.yaml st
 
 ### Clustering / High Availability
 
-Currently it is possible to specify the number of `replicas` but the implementation is naive.
+Currently it is possible to configure this chart to run multiple Traefik instances using
+either the `replicas` option or the `daemonset.enabled` option.  Enabling either results
+in a naive Traefik implementation.
```

Comment:
This sounds too scary... I would qualify that the part that is naive is, specifically, the approach to scaling.